### PR TITLE
font tweaks for commands

### PR DESCRIPTION
**Pull request by coinzdude**

### CODE REVIEW

File: packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx [hunk 1]
```diff
@@ -132,8 +132,8 @@ const CommandRow: React.FC<CommandRowProps> = ({
         </IconButton>
       }
       sx={{
-        paddingTop: comment ? 0 : 2,
-        paddingBottom: commandState.message ? 0 : 2,
+        paddingTop: comment ? 0 : 1,
```
Comment (toddtarsi) [inline packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx]: Let's remove this from here and instead add these as marginTop and marginBottom around line 165

Comment (toddtarsi) [reply packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx]: Also, can you add a marginTop of 1 to line 144 or so

File: packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx [hunk 2]
```diff
@@ -168,18 +168,16 @@ const CommandRow: React.FC<CommandRowProps> = ({
...
-          </Typography>
+          {camelToTitleCase(commandText)} {isDisabled ? '[Disabled]' : ''}
         </Box>
-        <Box sx={{ flex: 2, ...argTextFormat }}>
-          <Typography>{target}</Typography>
+         <Box sx={{ flex: 2, ...argTextFormat }}>
+          {target}
         </Box>
         <Box sx={{ flex: 2, ...argTextFormat }}>
-          <Typography>{value}</Typography>
```
Comment (toddtarsi) [inline packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx:182]: Nice work collapsing the styling into the parent div.

File: packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx [hunk 3]
```diff
@@ -190,7 +188,7 @@ const CommandRow: React.FC<CommandRowProps> = ({
       )}
       <Box
         className="fill pos-abs o-25"
-        sx={{ bgcolor, marginTop: -2, pointerEvents: 'none', zIndex: 75 }}
+        sx={{ bottom:'none', borderBottom:'none', borderBottomStyle:'none', bgcolor, pointerEvents: 'none', zIndex: 75 }}
```
Comment (toddtarsi) [inline packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx]: I think all of this border stuff can go. This div has no border by default. It's just a div with absolute positioning, low opacity, high z index, and all mouse events disabled. It's a transparent overlay

File: packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx (hunk 1 above)
Comment (toddtarsi) [reply packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx]: Alright, one more thing. Can you add a marginBottom of 1 to line 185, basically the commandState.message div

Comment (coinzdude) [reply packages/selenium-ide/src/browser/windows/ProjectEditor/components/Tests/TestCommandRow.tsx]: Done!

**Review by toddtarsi — APPROVED** [review]
@coinzdude - LGTM 👍

**Pull request merged**